### DISCUSSION TITLE
How to delete reblog or re-toot or repost or whatever it is?

### DISCUSSION THREAD

**Discussion by steve02476** [mastodon/mastodon#20331] [Q&A] [answered]
If I make a post and later wish to I can delete it. But I just reblogged/retooted something by accident and can’t figure out how to get that off my timeline. Of course I don’t want to delete the original, just the fact that I reposted it. Any advice? Using iOS app.

**Comment by trwnh** [comment, marked as answer]
tap the icon again to get it back to the inactive state

**Comment by steve02476** [comment]
Ah you mean the little circular “reblog “ icon? That did it, thanks. Never occurred to me.

Your help is much appreciated!
